src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/EJSONWrapper.java

```diff
@@ -118,4 +119,18 @@ private static byte[] bytesFromByteBuffer(ByteBuffer buffer) {
     buffer.get(bytes);
     return bytes;
   }
+
+  // Returns the float array value from the binary EJSON wrapper
+  public float[] getVectorValueForBinary() {
```

Comment:
This makes sense for immediate usage. In future may want to further refactor to expose just binary value, separate decoding of Vector value from binary to `float[]` (or `CqlVector`)